devtools/conda-envs/test.yaml

```diff
@@ -4,7 +4,7 @@ channels:
 dependencies:
   # Core
   - python
-  - cython =0.29.36
+  - cython
```

Comment:
I checked the logs and 3.x is pulled down, but just to be safe (and explicit about dependencies)
```suggestion
  - cython =3
```

Reply:
Done.

Reply:
Ah, this actually doesn't work and it should be `~3`.